src/LinearSolvers/bicgstab.jl

```diff
@@ -7,7 +7,7 @@ Van der Vorst, Henk A.
 of nonsymmetric linear systems."
 SIAM Journal on scientific and Statistical Computing 13, no. 2 (1992): 631-644.
 """
-function bicgstab(A, b, P, xi, to::TimerOutput;
+function bicgstab(A, b, P, xi;
```

Comment:
indeed, I think it is better to remove `TimerOutput` there. In my observations, it induces a small but non negligible slowdown

Reply:
Yes, that was my thought.